service/acme.go

```diff
@@ -86,6 +86,74 @@ func (s *acmeService) Trigger(id string) {
 
 }
 
+func (s *acmeService) Renew(c *model.Certificate) {
+	config := lego.NewConfig(c)
+
+	// This CA URL is configured for a local dev instance of Boulder running in Docker in a VM.
+	config.CADirURL = "https://acme-v02.api.letsencrypt.org/directory"
```

Comment:
would be cool to break out this URL into a separate config file or something so if it changes to v03 we won't have to search through a bunch of code